suite-common/icons/generateIconFont.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-console */
+// disable eslint import for this file because this whole file is using only devDependencies
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { generateFonts, FontAssetType, OtherAssetType } from 'fantasticon';
+import fs, { constants } from 'fs';
+import path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chalk from 'chalk';
+
+import { type IconName as AllAvailableIcons } from './src/icons';
+import { MOBILE_ICON_FONT_NAME } from './src/constants';
+
+const usedIcons = [
+    'arrowDown',
+    'arrowDown',
+    'arrowLineDown',
+    'arrowLineUp',
+    'arrowLineUpRight',
+    'arrowRight',
+    'arrowsCounterClockwise',
+    'arrowSquareOut',
+    'arrowUp',
+    'arrowUpRight',
+    'arrowUpRight',
+    'arrowURightDown',
+    'backspace',
+    'bookmark',
+    'bugBeetle',
+    'calendar',
+    'caretCircleRight',
+    'caretDown',
+    'caretDown',
+    'caretDownFilled',
+    'caretLeft',
+    'caretRight',
+    'caretUp',
+    'caretUpDown',
+    'caretUpFilled',
+    'change',
+    'check',
+    'checkCircle',
+    'checkCircleFilled',
+    'checks',
+    'circleDashed',
+    'clockClockwise',
+    'coins',
+    'coinVerticalCheck',
+    'copy',
+    'database',
+    'detective',
+    'discover',
+    'eye',
+    'eyeSlash',
+    'facebookLogo',
+    'filePdf',
+    'fingerprint',
+    'fingerprintSimple',
+    'flag',
+    'flagCheckered',
+    'gear',
+    'githubLogo',
+    'house',
+    'image',
+    'info',
+    'lifebuoy',
+    'lightbulb',
+    'link',
+    'lock',
+    'magnifyingGlass',
+    'palette',
+    'password',
+    'pencil',
+    'pencilSimpleLine',
+    'piggyBank',
+    'piggyBankFilled',
+    'plus',
+    'plusCircle',
+    'qrCode',
+    'question',
+    'question',
+    'question',
+    'question',
+    'shareNetwork',
+    'shieldWarning',
+    'shuffle',
+    'stack',
+    'swap',
+    'trashSimple',
+    'treeStructure',
+    'trezorLogo',
+    'trezorModelOne',
+    'trezorModelOneFilled',
+    'trezorModelT',
+    'trezorModelTFilled',
+    'trezorSafe3',
```

Comment:
Not as part of this PR, but it would be great to use internal names for file names for Trezor models (T2B1 instead of modelOne and so on). It's then easier to search for them, easier mapping and so on.

Reply:
Figma is source of truth here, I don't want to clash with it because it would require manual work everytime when updating icon from Figma in future. For us it's abstracted away in DeviceIcon.tsx anyway which should use correct model ids.

Reply:
Agree, internal names should be ideally used directly in Figma.